src/settings.qml

```diff
@@ -1005,6 +1005,10 @@ import QtQuick.Dialogs 1.0
             // from version 2.18.3
             property bool proform_performance_400i: false
 
+            // from version tbd
+            property string csafe_elliptical_port: ""
```

Comment:
they must be at the bottom of the others, otherwise the settings file will be corrupted @marcelrv

Reply:
ah.. yes, I had merge conflict each time, so I left it a bit above so not impacted by each version you made...
fixed now